docs/server/source/appendices/run-with-docker.md

```diff
@@ -90,6 +90,10 @@ docker run \
 
 ### Run BigchainDB
 
+To ensure that BigchainDB connects to the backend database bound to the virtual
+interface `172.17.0.1`, we modify the BigchainDB config file database.host parameter from
+`localhost` to `172.17.0.1` before starting the BigchainDB container.
+
```

Comment:
This section is titled "Run BigchainDB" --- configuration was done in an earlier section. Maybe move this paragraph to the end of the section titled "Configuration".

It's a bit hard to follow this sentence. "the BigchainDB config file database.host parameter" is difficult to parse. How about replacing everything from "interface `172.17.0.1`" with:

```text
interface `172.17.0.1`, you must edit the BigchainDB configuration file (`~/bigchaindb_docker/.bigchaindb`) and change database.host from `localhost` to `172.17.0.1`.
```